Firestore/core/src/util/log_stdio.cc

```diff
@@ -32,15 +32,15 @@ std::atomic<LogLevel> g_log_level(kLogLevelNotice);
 }  // namespace
 
 void LogSetLevel(LogLevel level) {
-  g_log_level.store(level);
+  g_log_level = level;
```

Comment:
Consider also using assignment-like syntax for initialization (at line 30 above).